src/views/product-downloads-view/boundary/server.ts

```diff
@@ -0,0 +1,61 @@
+// Data
+import boundaryData from 'data/boundary.json'
+// Server
+import { generateStaticProps as generateReleaseStaticProps } from 'lib/fetch-release-data'
+import { generateGetStaticProps } from 'views/product-downloads-view/server'
+// Types
+import { GetStaticProps } from 'next'
+import { ProductData } from 'types/products'
+import { ProductDownloadsViewStaticProps } from 'views/product-downloads-view/types'
+import { DesktopClientProps } from './components/desktop-client-callout/types'
+
+/**
+ * The Desktop Client CTA uses separate release data, fetched with this slug.
+ */
+const DESKTOP_CLIENT_RELEASE_SLUG = 'boundary-desktop'
+
+/**
+ * Boundary downloads static props extend the base product downloads view props,
+ * with props for the desktop client CTA.
+ */
+interface BoundaryDownloadsPageProps extends ProductDownloadsViewStaticProps {
+	desktopClientProps: DesktopClientProps
+}
+
+/**
+ * Get static props for the Boundary downloads page.
+ */
+const getStaticProps: GetStaticProps<BoundaryDownloadsPageProps> = async () => {
```

Comment:
We need [the release data for `boundary-desktop`](https://releases.hashicorp.com/boundary-desktop/index.json) to render the callout. This custom `getStaticProps` builds on the base product-downloads-view work, and adds in that extra data we need for Boundary.